### PR TITLE
Use cardinal hooks for avatar to remove white loading flash

### PR DESCRIPTION
**Pull request by 0xavro**
Before:
avatar-load-before

After:
avatar-load-after

### CODE REVIEW

**Review by 0xkokoro — APPROVED** [review]
lgtm :shipit:

File: packages/dialect-react-ui/components/CardinalAvatar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  const { addressImage, loadingImage } = useAddressImage(connection, address);
+
+  if (!address) return <></>;
+  return loadingImage ? (
+    <div
+      style={{
+        ...style,
+        height,
+        width,
+        borderRadius: "50%",
```
Comment (Kiryous) [inline packages/dialect-react-ui/components/CardinalAvatar/index.tsx:31]: To be consistent on styles approach — it's better to use tailwind styles here, like `className="dt-rounded-full"`

File: packages/dialect-react-ui/components/Avatar/index.tsx [hunk 2]
```diff
@@ -34,7 +33,7 @@ export default function Avatar({ publicKey, size = 'regular' }: PropTypes) {
     >
       <div className={`${textStyleMap[size]}`}>
         {program?.provider.connection ? (
-            <AddressImage
+            <CardinalAvatar
               height="100%"
```
Comment (Kiryous) [inline packages/dialect-react-ui/components/Avatar/index.tsx:37]: Same with the `height, width` props — to be consistent on styles approach — I thinks it's better to pass className prop to underlying component, and use tailwind styles here, like className="dt-h-full"

**Review by Kiryous — CHANGES_REQUESTED** [review]
Great job! Suggested to refactor styles a bit to use tailwind styles

**Comment by 0xavro** [comment]
@Kiryous I went with the tailwindcss changes instead of the raw styling

**Comment by 0xavro** [comment]
@Kiryous let's merge this into https://github.com/dialectlabs/react/pull/62 and review everything there

**Comment by 0xavro** [comment]
@Kiryous let's merge this into https://github.com/dialectlabs/react/pull/62 and review everything there

**Pull request merged**